activation/atxdb.go

```diff
@@ -42,6 +42,14 @@ func getAtxBodyKey(atxID types.ATXID) []byte {
 	return []byte(fmt.Sprintf("b_%v", atxID.Bytes()))
 }
 
+func getAtxCoinbaseKey(atx *types.ActivationTx) []byte {
+	return []byte(fmt.Sprintf("cb_%v_%v", atx.Coinbase.Bytes(), atx.PubLayerID.Bytes()))
```

Comment:
I prefer reusing the code here because it makes the intention clearer.
```suggestion
	return []byte(fmt.Sprintf("%v_%v", getCoinbaseAtxPrefix(atx.Coinbase), atx.PubLayerID.Bytes()))
```

Reply:
Hey Noam, i appreciate the suggestion, it makes a lot of sense. I'll make the changes as suggested and update usage in the API service. Also, i'd love to learn how to better spot ideas like this - is there any resource/do you have any resource to recommend that will help me develop this skillset?